MSAL/MSAL.xcodeproj/project.pbxproj

```diff
@@ -42,7 +42,6 @@
 		04A6B5D2226937890035C7C2 /* MSALRedirectUriVerifier.m in Sources */ = {isa = PBXBuildFile; fileRef = B21E07BC210E5458007E3A3C /* MSALRedirectUriVerifier.m */; };
 		04A6B5D5226937930035C7C2 /* MSALTelemetry.m in Sources */ = {isa = PBXBuildFile; fileRef = 96D9A5471E4AB22900674A85 /* MSALTelemetry.m */; };
 		04A6B5D6226937940035C7C2 /* MSALTelemetry.m in Sources */ = {isa = PBXBuildFile; fileRef = 96D9A5471E4AB22900674A85 /* MSALTelemetry.m */; };
-		04A6B5D7226937980035C7C2 /* (null) in Sources */ = {isa = PBXBuildFile; };
```

Comment:
can you also remove other 3 (null) lines :)?

Reply:
done